website/app/routes/show.js

```diff
@@ -128,7 +155,7 @@ export default class ShowRoute extends Route {
           });
         }
 
-        return {
+        this.currentModelCache = {
```

Comment:
Consider having this as an object, instead of a single variable whose content is constantly replaced (and so, lost) every time the user goes to another page:
```suggestion
        this.currentModelCache[path] = {
```
In this way you can also remove the `activate()` logic, which is not needed anymore.